### PR TITLE
Update Swedish.md with links in News and Literature

### PR DESCRIPTION
**Pull request by dearprakash**

### CODE REVIEW

File: living/swedish.md [hunk 1]
```diff
@@ -19,6 +23,14 @@ Culture / History:
 Literature:
 - https://litteraturbanken.se/ (Literature Bank makes available large collection of fiction and other text)
 - https://www.boktugg.se/ (Blog about news, trends, authors, publishers and lot more about bookgs in general)
+- https://varldslitteratur.se/ (Wesbite with news and information about author biographies, translated works, poems and novels)
+- https://www.svelitt.se/index.html (Website of Swedish literature society that publishes yearbooks containing research on Swedish and other Nordic literature)
+- https://swedishbookreview.org/ (Home page of Swedish Book Review presenting Swedish-language literature to the English-speaking world, through feature articles, translated extracts, interviews, book reviews and more)
+- https://www.sls.fi/sv/ (Home page of SLS that preserves, explore and spread knowledge about Swedish culture in Finland )
+- https://www.mitrania.se/ (Literary newspaper for and about Fantasy, Science fiction and horror)
+- https://bokforlagetatlas.se/ (Atlas is a progressive publisher with its feet in the present, but with an eye on the future)
```
Comment (thunderpoot) [inline living/swedish.md:31]: Please change this description to something more neutral, such as:
> Swedish publishing house publishing fiction, non-fiction, and reportage

File: living/swedish.md [hunk 2]
```diff
@@ -19,6 +23,14 @@ Culture / History:
...
+- https://www.dagensarena.se/essa/ (Dagens Arena publishes editorial articles on a radical, progressive basis of ideas)
+- https://dsm.nu (To say what is rarely said but should be said... needs to be said)
```
Comment (thunderpoot) [inline living/swedish.md:33]: This description is the direct English translation of the website's tagline, which is subjective and could be interpreted as biased.  Please change this to something more neutral, such as:
> The website of the magazine DSM, focusing on debate and media criticism.

**Review by thunderpoot — CHANGES_REQUESTED** [review]
Thanks for the contributions, @dearprakash.  Some items need slight adjustment to ensure impartiality. Please revise to be more objective, such as the suggested alternatives provided.

File: living/swedish.md (hunk 1 above)
Comment (dearprakash) [reply living/swedish.md]: Agreed and corrected

**Review by thunderpoot — APPROVED** [review]
Thank you for making those adjustments!

**Pull request merged**